File: Group 85-b/level 39/homework/hw1.py
##2. იგივე თაფლის გამოყენებით, გამოიტანეთ კენტი და ლუწი რიცხვები
tuple = (1, 2, 3, -4, 50, -6, 89, -100, 700, -3, 0.5, 0)
even_numbers = []
odd_numbers = []
for num in tuple:
    if num % 2 == 0:
        even_numbers.append(num)
    else:
        odd_numbers.append(num)
print("Even numbers:", even_numbers)
print("Odd numbers:", odd_numbers)

File: Group 85-b/level 59/homework/hw1.js
//გაქვს რიცხვების მასივი $[1, 15, 8, 22, 5, 30]$. გამოიტანე მხოლოდ ის რიცხვები, რომლებიც 10-ზე მეტია.

let numbers = [1, 15, 8, 22, 5, 30];
let result = [];
for (let i= 0; i < numbers.length; i++){
    if (numbers[i] > 10){
        result.push(numbers[i]);
    }
        
    }
    console.log(result);
    